great_expectations/core/_docs_decorators.py

```diff
@@ -285,16 +280,17 @@ def _add_text_below_string_docstring_argument(
 
     # Returns: includes an additional ":\n" that we need to strip out.
     if parsed_docstring.returns:
-        parsed_docstring.returns.description = (
-            parsed_docstring.returns.description.strip(":\n")
-        )
+        if parsed_docstring.returns.description:
+            parsed_docstring.returns.description = (
+                parsed_docstring.returns.description.strip(":\n")
```

Comment:
Should this stripped value be a constant? Is it ever OS dependent or consistent across environments?

Reply:
This is mainly to clean up the rendered output in our docs build process, so we set up and manage the environment (Netlify). This is a good question, I think if we were expecting to build in different environments we would address. For now I'll leave it as-is.